Makefile

```diff
@@ -50,23 +50,36 @@ config:  ## Create sample configuration files related to Snowflake, Amazon and G
 		cat .github/ci-test-connections.yaml > test-connections.yaml && \
 		echo "Please, update test-connections.yaml with your credentials")
 
-setup: config virtualenv install ## Setup a local development environment
+setup-dev-env: create-config create-venv install-dependency ## Setup a local development environment
 
-quality:
+run-precommit:
 	@$(PRECOMMIT) run --all-files
 
-test: virtualenv config ## Run all tests (use option: db=[db] run only run database-specific ones)
+run-tests: create-venv create-config ## Run all tests (use option: db=[db] run only run database-specific ones)
 ifdef db
 	@$(PYTEST) -s --cov --cov-branch --cov-report=term-missing -m "$(db)"
 else
 	@$(PYTEST) -s --cov --cov-branch --cov-report=term-missing
 endif
 
-unit: virtualenv config ## Run unit tests
+run-unit-tests: create-venv create-config ## Run unit tests
 	@$(PYTEST) -s --cov --cov-branch --cov-report=term-missing -m "not integration"
 
-integration: virtualenv config  ## Run integration tests
+run-integration-tests: create-venv create-config  ## Run integration tests
 	@$(PYTEST) -s --cov --cov-branch --cov-report=term-missing -m integration
 
-help:
-	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+logs: ## View logs of the all the containers
+	docker-compose -f dev/docker-compose.yaml logs --follow
+
+stop: ## Stop all the containers
+	docker-compose -f dev/docker-compose.yaml down
+
+clean: ## Remove all the containers along with volumes
+	docker-compose -f dev/docker-compose.yaml down  --volumes --remove-orphans
+	rm -rf dev/logs
+
+build-run: ## Build the Docker Image & then run the containers
+	docker-compose -f dev/docker-compose.yaml up --build -d
+
+help: ## Prints this message
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-41s\033[0m %s\n", $$1, $$2}'
```

Comment:
Would it be possible to introduce the new Makefile tasks without renaming all the existing ones? 
It is usually handy to have short-named tasks, and I believe the introduction to container operations shouldn't discourage the other commands

Reply:
Created a new makefile please let me know if you have a better name for it and also how it is looking now.